src/number_theory/number_field/embeddings.lean

```diff
@@ -245,6 +246,87 @@ subtype.ext (w.2).some_spec
 
 lemma pos_iff (w : infinite_place K) (x : K) : 0 < w x ↔ x ≠ 0 := absolute_value.pos_iff w.1
 
+lemma conjugate_eq (φ : K →+* ℂ) :
+  mk (complex_embedding.conjugate φ) = mk φ :=
+begin
+  ext x,
+  convert congr_fun (congr_arg coe_fn (complex_embedding.place_conjugate φ)) x,
```

Comment:
Does `exact` work here or do we really need `convert`?

Reply:
Thanks for the review. Indeed, it should be an `exact`